### PR TITLE
feat: enable monitoring solution

### PR DESCRIPTION
**Pull request by AchimHaaser1**
Description:
Deploys IBM monitoring Instance plus an intel VSI to collect the metrics.

- [x] Breaking Change Major release

##### Release notes content

**Breaking Change**:
- New landing zone version `6.4.0` creates VSis using `VNIs` and legacy interfaces are not used any more.
- Landing zone preset reworked to create security groups separately and then assign to VSIs. This will recreate the VSIs.

**Features:**
- Provision Monitoring Instance and an intel VSI to host and process monitoring metrics for SAP
- Upgrade Rhel images running on Intel VSI (jump box and network services vsi to `RHEL 9.4`)
- Upgraded minimum IBM power linux sap ansible role version to `v3.0.0`
- Upgraded IBM TF version to `1.72.3`

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Review by surajsbharadwaj — APPROVED** [review]

**Review by surajsbharadwaj — DISMISSED** [review]

File: modules/powervs-vpc-landing-zone/README.md [hunk 1]
```diff
@@ -4,8 +4,9 @@
 
 This module provisions the following resources in IBM Cloud:
 - A **VPC Infrastructure** with the following components:
-    - One VSI for one management (jump/bastion) VSI,
+    - One VSI for management (jump/bastion) VSI,
```
Comment (ludwig-mueller) [inline modules/powervs-vpc-landing-zone/README.md:7]: There's a VSI too many in this sentence.

**Review by ludwig-mueller — APPROVED** [review]

**Review by surajsbharadwaj — APPROVED** [review]

**Pull request merged**